### PR TITLE
Use Node v12.13.0 (LTS) for development

### PR DESCRIPTION
**Pull request by etpinard**
[Node v12](https://nodejs.org/en/blog/release/v12.13.0/) is now LTS, so let's start using it.

Note the node-v12 docker containers provided by CircleCI ship with Chrome v78. This makes some of our jasmine tests fail. 

----

The failing tests are:

https://github.com/plotly/plotly.js/blob/742b83da9f851860d3f217c7fb56b763912b541b/test/jasmine/tests/hover_label_test.js#L2527-L2551

This :arrow_up: one can probably be fixed be tweaking the pixels values in the assertions.

https://github.com/plotly/plotly.js/blob/742b83da9f851860d3f217c7fb56b763912b541b/test/jasmine/tests/colorbar_test.js#L400-L460

This :arrow_up: one is odd. Something might be up with `{editable: true}` colorbar drag interactions?

https://github.com/plotly/plotly.js/blob/742b83da9f851860d3f217c7fb56b763912b541b/test/jasmine/tests/splom_test.js#L946-L998

This :arrow_up: one might means that one the most painful workaround in the library got fixed (see https://github.com/plotly/plotly.js/pull/2939) 

-----

@plotly/plotly_js I won't have time to fixup the jasmine tests before my vacation. This is not at all a high priority issue, but if you're interested in getting this merged, feel free to push to this branch.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "start-image_viewer": "node devtools/image_viewer/server.js",
     "start": "npm run start-test_dashboard",
     "baseline": "node tasks/baseline.js",
-    "preversion": "check-node-version --node 10 --npm 6 && npm-link-check && npm ls --prod",
+    "preversion": "check-node-version --node 12.13 --npm 6.12.0 && npm-link-check && npm ls --prod",
```
Comment (etpinard) [inline package.json]: @archmoj I chose slightly more strict version checks this time.

Comment (etpinard) [reply package.json]: Now requesting for npm `6.13.x` https://github.com/plotly/plotly.js/pull/4322/commits/7759c7602be9d0247176d817bddf6ac0f6b5e630

**Comment by etpinard** [comment]
@archmoj I got the test to pass on CI and locally. Tagging as `reviewable` !

**Comment by archmoj** [comment]
Thanks very much @etpinard 
:dancer:

**Pull request merged**